### PR TITLE
feat: add variable to configure ebs optimization for runner instances

### PR DESCRIPTION
**Pull request by AlexShemeshWix**
feat: add variable to configure ebs optimization for runner instances

### CODE REVIEW

**Comment by AlexShemeshWix** [comment]
#skipreview

File: main.tf [hunk 1]
```diff
@@ -288,6 +288,7 @@ module "runners" {
   pool_lambda_reserved_concurrent_executions = var.pool_lambda_reserved_concurrent_executions
 
   ssm_housekeeper = var.runners_ssm_housekeeper
+  ebs_optimized = var.runners_ebs_optimized
```
Comment (npalm) [inline main.tf:291]:
```suggestion
  ebs_optimized  = var.runners_ebs_optimized
```

Comment (npalm) [reply main.tf:291]: please can you run the terraform formatter?

**Review by npalm — CHANGES_REQUESTED** [review]
Thanks, looks good, please can you fix the formatting issues?

File: main.tf (hunk 1 above)
Comment (AlexShemeshWix) [reply main.tf]: Done

**Review by npalm — APPROVED** [review]

**Pull request merged**